### PR TITLE
Changing zookeeper configuration to accept the values as per kafka documentation

### PR DESCRIPTION
**Pull request by ajaysarda**
Tested locally and on centralized remote zookeeper+kafka instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Listening for messages coming from a kafka broker.
 ```json
 {
   "consumer" : {
-    "zookeeper.host" : "<zookeeperHost>",
+    "zookeeper.connect" : "<zookeeperConnectionString>",
     "port": "<zookeeperPort>",
```
Comment (jtruelove) [inline README.md:33]: missed this, is this port line needed now? I'm guessing no.

**Comment by jtruelove** [comment]
I think I missed one small thing on your change.

File: README.md (hunk 1 above)
Comment (ajaysarda) [inline README.md:33]: no, Should I send you another one or wait till next one (publish to multiple topics)?

Comment (jtruelove) [inline README.md:33]: your call

**Pull request merged**